Assets/MixedRealityToolkit.Tests/PlayModeTests/BoundsControlTests.cs

```diff
@@ -0,0 +1,314 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#if !WINDOWS_UWP
+// When the .NET scripting backend is enabled and C# projects are built
+// The assembly that this file is part of is still built for the player,
+// even though the assembly itself is marked as a test assembly (this is not
+// expected because test assemblies should not be included in player builds).
+// Because the .NET backend is deprecated in 2018 and removed in 2019 and this
+// issue will likely persist for 2018, this issue is worked around by wrapping all
+// play mode tests in this check.
+
+using Microsoft.MixedReality.Toolkit.UI;
+using NUnit.Framework;
+using System.Collections;
+using UnityEngine;
+using UnityEngine.TestTools;
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System.Linq;
+using Assert = UnityEngine.Assertions.Assert;
+using Microsoft.MixedReality.Toolkit.UI.Experimental;
+
+namespace Microsoft.MixedReality.Toolkit.Tests
+{
+    /// <summary>
+    /// TODO: This test still needs to be adjusted 
+    /// Currently it's just a copy of whatever was tested in Bounding Box
+    /// </summary>
+    public class BoundsControlTests
+    {
+        #region Utilities
+        [SetUp]
+        public void Setup()
+        {
+            PlayModeTestUtilities.Setup();
+        }
+
+        [TearDown]
+        public void ShutdownMrtk()
+        {
+            PlayModeTestUtilities.TearDown();
+        }
+
+        /// <summary>
+        /// Instantiates a bounds control at 0, 0, -1.5f
+        /// box is at scale .5, .5, .5
```

Comment:
the scale and position vectors should be saved and then re-used across these tests.